source/clear-linux/tutorials/multi-boot/multi-boot-sles.rst

```diff
@@ -87,15 +84,15 @@ Install SUSE\* Linux Enterprise 12 SP2
 
       .. note:: The root partition UUID used below is unique to this example.
 
-         .. code-block:: console
+         .. code-block:: bash
 
-            # cd /boot/efi/loader/entries
+            sudo cd /boot/efi/loader/entries
 
-            # vi suse.conf
+            sudo vi suse.conf
```

Comment:
Let's consolidate lines 89 and 91 to "sudoedit /boot/efi/loader/entries/suse.conf"